shared/src/commonMain/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -32,47 +31,37 @@ class AuthRepository(
 	}
 
 	suspend fun userProfile() =
```

Comment:
`suspend` is redundant from now, I'll get rid of it in upcoming PR to not introduce more changes because it will affect its usages in `ProfileViewModelTest`